USBIRToy.v23.hex

```diff
@@ -0,0 +1,2071 @@
+:020000040000FA
```

Comment:
Is there a persistent place we can pull this file from, such that we don't commit it directly to our repo?